src/components/InstanceFilters/InstanceFiltersBrowse/InstanceFiltersBrowse.js

```diff
@@ -140,6 +146,25 @@ const InstanceFiltersBrowse = props => {
             </Accordion>
           )}
           <Accordion
+            closedByDefault
+            label={intl.formatMessage({ id: `ui-inventory.filters.${FACETS.HELD_BY}` })}
+            id={FACETS.CALL_NUMBERS_HELD_BY}
+            name={FACETS.CALL_NUMBERS_HELD_BY}
+            separator={false}
+            header={FilterAccordionHeader}
+            displayClearButton={activeFilters[FACETS.CALL_NUMBERS_HELD_BY]?.length > 0}
+            onClearFilter={() => onClear(FACETS.CALL_NUMBERS_HELD_BY)}
+          >
+            <CheckboxFacet
+              name={FACETS.CALL_NUMBERS_HELD_BY}
+              dataOptions={facetsOptions[FACETS_OPTIONS.HELD_BY_OPTIONS]}
+              selectedValues={activeFilters[FACETS.CALL_NUMBERS_HELD_BY]}
+              isPending={getIsPending(FACETS.CALL_NUMBERS_HELD_BY)}
+              onChange={onChange}
```

Comment:
Please also add `onFetch={handleFetchFacets}` prop to re-fetch facets when changing call numbers option

Reply:
`onFetch` is called only when we click on the `+More` button or when we place the cursor in a facet search box field.